app/views/collections/show.html.slim

```diff
@@ -13,12 +13,19 @@
         - else
           i.fa-solid.fa-lock.lock-icon
           =< t('.visibility.private')
-      span
-        i.fa-solid.fa-users
-        - if @collection.users.size == 1
-          =< t('.no_other_user')
-        - else
-          =< t('.num_of_other_users', count: @collection.users.size - 1)
+      - if action_name != 'view_shared'
+        span
+          i.fa-solid.fa-users
+          - if @collection.users.size == 1
+            =< t('.no_other_user')
+          - else
+            =< t('.num_of_other_users', count: @collection.users.size - 1)
+        span
+          i.fa-regular.fa-envelope
+          - if @num_of_invites == 0
```

Comment:
```suggestion
          - if @num_of_invites.zero?
```

I tend to prefer `zero?` (similar to `empty?`) for readability over `== 0` unless `0` is meant as some math value, i.e. coordinate or whatever.